integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This tests core use cases for u-root such as:
```

Comment:
u-root or webbot?

Reply:
This should be webboot. We copied the interrogation package over from u-root for webboot's integration testing, but we did not catch the u-root there. We have changed the name since then.